_data/jobs.yml

```diff
@@ -0,0 +1,34 @@
+- name: Full Stack Engineer - DCI
+  url: http://bit.ly/FSE-DCI
+  image_url: ../images/jobs/dci.jpg
```

Comment:
Since we are committing images to GIT, I'm getting weary of the GIT size in the long run 😅. 

Is there a way to avoid committing images binary files to GIT? Maybe host the images in  https://cloudinary.com?

Reply:
good point!

Reply:
ok mas @firewalker06 , do we have a cloudinary account ? or maybe i'll create one ?

Reply:
hi @azioarv you can just create one and share the credentials in private channel to us later

Reply:
Hi ! i've created cloudinary account, sorry for the slow progress :pensive: , for the credentials, i'll share in the private message soon